views/random-beer.hbs

```diff
@@ -0,0 +1,11 @@
+
+    {{#each responseFromApi}}
+    <div>
+        <img src="{{this.image_url}}">
+        {{this.name}}
+        {{this.description}}
+        {{this.tagline}}
+        {{this.food_pairing}}
```

Comment:
since foor_pairing is another array you could have use another # each to list the elements.